tst/EnergyPlus/unit/StandardRatings.unit.cc

```diff
@@ -1180,7 +1180,7 @@ TEST_F(EnergyPlusFixture, ChillerIPLVTestWaterCooled)
                     ObjexxFCL::Optional_int_const(),
                     ObjexxFCL::Optional<const Real64>());
 
-    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 5.44); // 18.56 IPLV
+    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 5.47); // 18.56 IPLV
```

Comment:
IPLV values change due to correction to the condenser entering water temperature per the AHRI STD.